doc/install.rst

```diff
@@ -0,0 +1,111 @@
+Installation
+============
+
+.. raw:: html
+
+    <style>
+        .breadcrumb {
+            display: none;
+        }
+        h1 {
+            text-align: center;
+            margin-bottom: 15px;
+        }
+        p.lead.grey-text {
+            margin-bottom: 30px;
+        }
+        .footer-relations {
+            border-top: 0px;
+        }
+        pre {
+            background-color: #FFFFFF00;
+            border-style: hidden;
+            font-size: 87.5% !important;
+            margin: 0 !important;
+            padding: 0;
+        }
+        code {
+            margin: 0 !important;
+            padding: 0 40px 0 40px !important;
+        }
+    </style>
+
+    <div class="container" id="main-column">
+        <div class="text-center">
+            <p class="lead grey-text w-responsive mx-auto">
+                FlamingPy requires Python 3.8 or newer.
+            </p>
+            <p class="lead grey-text w-responsive mx-auto mb-6">
+                If you currently do not have Python 3 installed, we recommend
+                <a href="https://www.anaconda.com/download/">Anaconda for Python 3</a>,
+                a distributed version of Python packaged for scientific computation.
+            </p>
+        </div>
+
+        <ul class="picker nav nav-pills nav-justified mt-6" id="version">
+            <li class="nav-item">
+                <a class="nav-link active" data-toggle="tab" href="#stable" role="tab">Stable</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" data-toggle="tab" href="#dev" role="tab">Source and Testing</a>
+            </li>
+        </ul>
+
+        <!-- Tab panels -->
+        <div class="tab-content pt-0" id="tab-version">
+            
+            <div class="tab-pane in show active" id="stable" role="tabpanel">
+                <pre>
+                    <code class="bash">
+    # Install the latest released version of FlamingPy, all in a single package.
+    pip install flamingpy
```

Comment:
```suggestion
    python -m pip install flamingpy
```

Reply:
Always liked specifying which "python" explicitly more. But if all Xanadu projects tend to use only "pip install" for this HTML section, I am happy to keep it as is.